scholarly/scholarly.py

```diff
@@ -223,7 +223,7 @@ def __init__(self, __data):
         else:
             self.id = re.findall(_CITATIONAUTHRE, __data('a')[0]['href'])[0]
             self.url_picture = _HOST+'/citations?view_op=medium_photo&user={}'.format(self.id)
-            self.name = __data.find('h3', class_='gsc_oai_name').text
+            self.name = __data.find('h3', class_='gs_ai_name').text
```

Comment:
I can confirm this change.

Reply:
This is now resolved in a more dynamic manner in #40